07LessonSeven/in-class-project/index.html

```diff
@@ -2,9 +2,89 @@
 <html>
 
 <head>
+
     <title>Template</title>
+
+    <link rel="stylesheet" href="css/my-style.css">
+
 </head>
 
-<body></body>
+<body>
+
+  <main>
+
+    <div class="my_image">
+
+      <h1>NASA</h1>
+
+    </div>
+
+    <div class="links">
+
+      <div id="social_media">
+
+        <h2>Social Media</h2>
+
+        <ul>
+
+          <li>
+            Twitter-<a href="nasatwitter">@nasa</a>
+          </li>
+
+          <li>
+            Facebook-<a href="nasafb">fb.com/nasa</a>
+          </li>
+
+          <li>
+            Instagram-<a href="nasainsta">@nasa</a>
+          </li>
+
+          <li>
+            Youtube-<a href="nasaYT">Youtube.com/NASA</a>
+          </li>
+
+        </ul>
+
+      </div>
+
+      <div id="office_info">
+
+        <h2>OFFICE INFO</h2>
+
+        <p>Public Communications Office</p>
+
+        <p>NASA Headquarters</p>
+
+        <p>300 E. Street SW. Suite 5R30</p>
+
+        <p>Washinton,DC 20546</p>
+
+        <p>(202)358-0001 (office)</p>
+
+        <p>(202)358-4338 (Fax)</p>
+
+      </div>
+
+      <div id="contact_us">

```

Comment:
this id name doesn't match the one in your css! also, since these three elements are very similar, I would suggest one classname for all of them or going through their parent element